### PR TITLE
EditPen: add localized text and title

### PR DESCRIPTION
**Pull request by wiese**
Make sure the edit link is self-explanatory even without resorting to
the icon.

Bug: T212206

Could use #150 for a working edge-to-edge test (if we insist on that).

### CODE REVIEW

**Comment by wiese** [comment]
Let's talk about the way the text is hidden, if you want. Am aware this may be controversial.

**Comment by wiese** [comment]
Rebased

**Comment by wiese** [comment]
Rebased

File: src/components/EditPen.vue [hunk 1]
```diff
@@ -36,5 +38,9 @@ export default class EditPen extends Vue {
 			transform: scaleX(-1);
 		}
 	}
+
+	&__edit-action-text {
+		@include sr-only();
```
Comment (jakobw) [inline src/components/EditPen.vue]: https://github.com/twbs/bootstrap/blob/v4-dev/scss/mixins/_screen-reader.scss#L6 Cool :)

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "@": "serverBuild/"
   },
   "dependencies": {
+    "bootstrap": "^4.2.1",
```
Comment (jakobw) [inline package.json]: I'm secretly a [bulma](https://bulma.io/documentation/) fan, but apparently bootstrap is cool again since v4.

Comment (wiese) [reply package.json]: I'm openly a [foundation](https://foundation.zurb.com/sites/docs/) fan but totally undogmatic.
Would be happy with either framework that does good for us (the bootstrap answer to `sr-only` looked nasty but elaborate). Either way I'd prefer if we could preserve SOLAC, hence the mix-in proposal.

File: src/components/EditPen.vue (hunk 1 above)
Comment (wiese) [reply src/components/EditPen.vue]: My IDE allows me to click that in `src/styles/_main.scss`

File: src/components/EditPen.vue [hunk 3]
```diff
@@ -19,9 +22,7 @@ import Sectionedit from './Sectionedit.vue';
 		href: String,
 	},
 } )
-export default class EditPen extends Vue {
-
-}
+export default class EditPen extends ( mixins( Messages ) as VueConstructor<MessagesMixin> ) {}
```
Comment (wiese) [inline src/components/EditPen.vue]: @bitPogo I removed the newline

Comment (bitPogo) [reply src/components/EditPen.vue]: :)

**Pull request merged**